### PR TITLE
Fix meshes examples

### PR DESCRIPTION
**Pull request by glazychev-art**
Closes: https://github.com/networkservicemesh/deployments-k8s/issues/6872
Signed-off-by: Artem Glazychev <artem.glazychev@xored.com>

NSM+Istio_Datapath

We have Workload-1 on the Cluster-1 and Workload-2 on the Cluster-2.
If we call `curl` from Workload1 to Workload2 network service, istio sometimes replaces the IP address with `NsmSrcIPs` and we come to ourselves (to Workload1, chain OUTPUT probably fires), but we should go to Workload-2.

### CODE REVIEW

**Comment by edwarnicke** [comment]
For reference on the 127.0.0.6 stuff: https://github.com/istio/istio/issues/29603

File: examples/interdomain/nsm_istio/nse-auto-scale/iptables-map.yaml [hunk 1]
```diff
@@ -2,9 +2,6 @@
 - -N NSM_PREROUTE
 - -A NSM_PREROUTE -j ISTIO_REDIRECT
 - -I PREROUTING 1 -p tcp -i {{ .NsmInterfaceName }} -j NSM_PREROUTE
-- -N NSM_OUTPUT
-- -A NSM_OUTPUT -j DNAT --to-destination {{ index .NsmSrcIPs 0 }}
-- -A OUTPUT -p tcp -s 127.0.0.6 -j NSM_OUTPUT
```
Comment (edwarnicke) [inline examples/interdomain/nsm_istio/nse-auto-scale/iptables-map.yaml:7]: OK... after some review, I think you are likely correct to remove this.  Do be extra careful to check that traffic coming into the istio pod from the istio-cluster does in fact get correctly proxied to the workload pod in the nsm-cluster.

Comment (glazychev-art) [reply examples/interdomain/nsm_istio/nse-auto-scale/iptables-map.yaml:7]: Of course, I already checked it. But I'll double check

**Comment by glazychev-art** [comment]
@edwarnicke 
Yes, istio-proxy can redirect traffic to 127.0.0.6. After that this traffic goes to OUTPUT chain (step 5 from https://jimmysong.io/en/blog/sidecar-injection-iptables-and-traffic-routing/) but we have the rules:
```
-N NSM_OUTPUT
-A NSM_OUTPUT -j DNAT --to-destination {{ index .NsmSrcIPs 0 }}
-A OUTPUT -p tcp -s 127.0.0.6 -j NSM_OUTPUT
```
which, as I understand it, will redirect traffic to the NSM_OUTPUT chain. And NSM_OUTPUT will change the destination address to NsmSrcIPs[0], that is, the traffic will return to its own pod (Workload-1).

**Comment by glazychev-art** [comment]
@edwarnicke 
You are absolutely right, we do not need to touch these rules.
I added some description to the example.

**Review by denis-tingaikin — APPROVED** [review]

**Pull request merged**